tests/accuracy.rs

```diff
@@ -126,11 +126,23 @@ fn test_cooley_tukey() {
         let signal = random_signal(len);
         assert!(ct_matches_dft(signal, false), "length = {}", len);
     }
+
+    //test some specific lengths > 100
+    for &len in &[256, 768] {
+        let signal = random_signal(len);
+        assert!(ct_matches_dft(signal, false), "length = {}", len);
+    }
 }
 
 #[test]
 fn test_cooley_tukey_inverse() {
-    for len in 1..100 {
```

Comment:
I changed it to match the non-inverse test function, which has been at 2 for a long time. But you're absolutely right, I should have changed the other one to be 1 instead of changing this one to be 2. I'll make a commit that fixes this.